.evergreen/find_cmake.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
This is a copy from master in order to build the latest C driver. All C++ branches going back to 3.3 test against the latest libmongoc. But the 3.2 branch was building 1.9.2. There were test failures due to a bug that was fixed in 1.11:

https://github.com/mongodb/mongo-c-driver/commit/806cddb2acde03f5462460491f38fa21332b7d97#diff-5853ab613d7df98e940febe8c1553fc6R370

Specifically, a default write concern could get included in a command. Here's an example of one failure:
https://evergreen.mongodb.com/task/cxx_driver_v3.2_ubuntu1604_debug_asan_compile_and_test_with_shared_libs_patch_c5a6e0a1cc2876eed025c01dc61a935d8d554c53_5ed70c362a60ed42a93e81e3_20_06_03_02_34_39

IMO since we test against latest C driver on all other branches I think it makes sense to do so for the 3.2 branch too. It's always safe for users to upgrade libmongoc, and this permits us to test older C++ branches against the latest server.